native/cocos/renderer/pipeline/custom/FrameGraphDispatcher.cpp

```diff
@@ -710,11 +728,14 @@ void buildBarriers(FrameGraphDispatcher &fgDispatcher) {
     auto *scratch = fgDispatcher.scratch;
     const auto &graph = fgDispatcher.graph;
     const auto &layoutGraph = fgDispatcher.layoutGraph;
-    const auto &resourceGraph = fgDispatcher.resourceGraph;
+    auto &resourceGraph = fgDispatcher.resourceGraph;
     auto &relationGraph = fgDispatcher.relationGraph;
     auto &externalResMap = fgDispatcher.externalResMap;
     auto &rag = fgDispatcher.resourceAccessGraph;
 
+    // disable gfx internal deduce
+    gfx::Device::getInstance()->enableAutoBarrier(false);
```

Comment:
Is it possible to configure when the render pipeline is activated?

Reply:
Yes, we can move this change to custom pipeline activation.